RELEASE_NOTES.md

```diff
@@ -1,3 +1,9 @@
+## 3.0.1
+### Update
+  - Added 2 catalog parameters in AssemblyUtilImpl
```

Comment:
Describe the parameters and what they do - no need to mention implementation details like in which file they're accessed

Reply:
Updated!

Reply:
:+1: